src/main/java/com/josdem/vetlog/command/UserCommand.java

```diff
@@ -38,6 +38,12 @@ public class UserCommand implements Command {
     @Size(min = 1, max = 100)
     private String lastname;
 
+    @Size(min = 1, max = 250)
+    private String phone;
+
+    @Size(min = 1, max = 250)
+    private String countryCode;
+
```

Comment:
Based on this information: https://countrycode.org/ max string lenght for country codes is 5, therefore 250 seems too large. However, no worries we will take care of this lenght in this issue: https://github.com/josdem/vetlog-spring-boot/issues/319

Reply:
That's certainly true. thank you. 
Please take action using 319